package.json

```diff
@@ -115,23 +115,24 @@
     "@babel/preset-typescript": "^7.18.6",
     "@babel/register": "^7.18.9",
     "@percy/cli": "^1.20.0",
-    "@percy/storybook": "^4.3.5",
+    "@percy/storybook": "^4.3.6",
     "@skyscanner/bpk-foundations-web": "^16.1.0",
     "@skyscanner/bpk-svgs": "^16.4.0",
-    "@skyscanner/stylelint-config-skyscanner": "^7.0.0",
-    "@storybook/addon-a11y": "^6.5.14",
-    "@storybook/addon-actions": "^6.5.14",
-    "@storybook/addon-viewport": "^6.5.14",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/react": "^6.5.14",
+    "@skyscanner/stylelint-config-skyscanner": "^10.0.0",
```

Comment:
Bumping `@skyscanner/stylelint-config-skyscanner` was needed as the SB migration caused some nested dependencies upgrades which resulted it incompatible versions of `prettier` and `eslint-plugin-prettier` being used. The latest versions of `@skyscanner/stylelint-config-skyscanner` and `@skyscanner/eslint-config-skyscanner` solves this issue.